internal/config/config.go

```diff
@@ -70,6 +70,7 @@ type Peer struct {
 
 	FilterIRR          *bool `yaml:"filter-irr" description:"Should IRR filtering be applied?" default:"false"`
 	FilterRPKI         *bool `yaml:"filter-rpki" description:"Should RPKI invalids be rejected?" default:"true"`
+	ForceRPKI          *bool `yaml:"force-rpki" description:"Should Only RPKI Valids be accepted?" default:"false"`
```

Comment:
Should this be called strict RPKI? Force RPKI sounds (to me at least) like it relates to forcing RPKI itself, like ignoring RTR errors, rather than something to do with filtering.